packages/codemods/src/codeMods/types.ts

```diff
@@ -26,3 +26,7 @@ export interface Codemod<T = SourceFile> {
    */
   enabled?: boolean;
 }
+
+export interface EnumMap {
```

Comment:
Not necessary, but if you have time you could change this to `export type Dictionary<T> = {[key: string]: T}` so that it can be used generically :D